dashboard/app/models/course_version.rb

```diff
@@ -89,4 +97,8 @@ def destroy_and_destroy_parent_if_empty
   def contained_lessons
     units.map(&:lessons).flatten
   end
+
+  def all_standards_url
+    content_root_type === 'UnitGroup' ? standards_course_path(content_root) : standards_script_path(content_root)
```

Comment:
Sorry, should've noticed this before but did you mean to use the triple equals here?

Reply:
Oops, no, this was just javascript on the brain. apparently it does the same thing most of the time, which I guess the same could be said for these two operators javascript. Anyway, nice catch, fixed!